frontends/CMakeLists.txt

```diff
@@ -99,97 +99,6 @@ set (P4_FRONTEND_SRCS
   p4/metrics/exportMetrics.cpp
   )
 
-set (P4_FRONTEND_HDRS
-  p4/actionsInlining.h
-  p4/alias.h
-  p4/callGraph.h
```

Comment:
Your disabling linting of all the frontend header files, not just the P4_14 ones?

Reply:
With the way the linters work nowadays (https://github.com/p4lang/p4c/blob/main/cmake/Linters.cmake#L1) these header collections are not used anymore. I am removing them to clear up some space.